src/Datadog.Trace.ClrProfiler.Managed/AutoInstrumentation/WebRequest/WebRequestGetRequestStreamIntegration.cs

```diff
@@ -0,0 +1,62 @@
+using System.Net;
+using Datadog.Trace.ClrProfiler.CallTarget;
+using Datadog.Trace.ExtensionMethods;
+
+namespace Datadog.Trace.ClrProfiler.AutoInstrumentation.WebRequest
+{
+    /// <summary>
+    /// CallTarget integration for WebRequest.GetRequestStream
+    /// </summary>
+    [InstrumentMethod(
+        AssemblyName = WebRequestCommon.NetFrameworkAssembly,
+        TypeName = WebRequestCommon.HttpWebRequestTypeName,
+        MethodName = MethodName,
+        ReturnTypeName = ClrNames.Stream,
+        ParameterTypeNames = new string[0],
+        MinimumVersion = WebRequestCommon.Major2,
+        MaximumVersion = WebRequestCommon.Major4,
+        IntegrationName = WebRequestCommon.IntegrationName)]
+    [InstrumentMethod(
+        AssemblyName = WebRequestCommon.NetCoreAssembly,
+        TypeName = WebRequestCommon.HttpWebRequestTypeName,
+        MethodName = MethodName,
+        ReturnTypeName = ClrNames.Stream,
+        ParameterTypeNames = new string[0],
+        MinimumVersion = WebRequestCommon.Major4,
+        MaximumVersion = WebRequestCommon.Major5,
+        IntegrationName = WebRequestCommon.IntegrationName)]
+    public class WebRequestGetRequestStreamIntegration
```

Comment:
nit: Can you add `Http` to the front of this type name and `WebRequestGetResponseIntegration`?

Reply:
Done